feature-libs/order/components/order-confirmation/order-guest-register-form/order-guest-register-form.component.ts

```diff
@@ -75,7 +78,8 @@ export class OrderGuestRegisterFormComponent implements OnDestroy {
       );
       if (
         !this.subscription &&
-        !this.featureConfigService.isEnabled('authorizationCodeFlowByDefault')
+        this.authConfigService.getOAuthFlow() ===
```

Comment:
`!authorizationCodeFlowByDefault` is the same as saying "do this for ROPC Grant only".